js/interface.js

```diff
@@ -1,28 +1,23 @@
 
 var widgetId = Fliplet.Widget.getDefaultId();
 var data = Fliplet.Widget.getData(widgetId) || {};
-var organizationId = Fliplet.Env.get('organizationId');
 var appId = Fliplet.Env.get('appId');
+var dataSourceProvider = null;
+var $dataColumnsEmail = $('select#dataSourceColumnEmail');
```

Comment:
`select` here is unnecessary, you can only have one element per unique ID.

Reply:
removed select, used only unique iD